### DISCUSSION TITLE
Installation without composer.

### DISCUSSION THREAD

**Discussion by manuvarghese236** [kreait/firebase-php#950] [General]
My problem is that Composer can't installed on my server. Is there any way to install it manually?

**Comment by jeromegamez** [comment]
You can install composer on your local computer, do the `composer install` there and then upload your project including the `vendor` directory.

Just make sure that the PHP version and needed PHP extensions are the same than on your server, otherwise you might run into problems.

If your computer has different versions, you could run `composer install` inside a Docker container with the same operating system, PHP version and extensions as on your server, instead of running it locally on your computer.
